src/functions/plot_clusters.py

```diff
@@ -88,15 +88,15 @@ def plot_clusters(clusters: Dict[str,dict],
     return fig, (ax1,ax2)
 
 
-def add_scatter_cluster(ax: matplotlib.axes.Axes, x: np.ndarray[float], y: np.ndarray[float], cov: np.ndarray[float], cluster_id = int, error_dir: str = "h") -> Tuple[matplotlib.axes.Axes, Any]:
+def add_scatter_cluster(ax: matplotlib.axes.Axes, x: np.ndarray, y: np.ndarray, cov: np.ndarray, cluster_id = int, error_dir: str = "h") -> Tuple[matplotlib.axes.Axes, Any]:
```

Comment:
Please keep the type hints. Some of these type hints have been removed in the previous pull request. It would be helpful if you can please add them back.